### ISSUE TITLE
Add ability to copy message references from Web App

### ISSUE THREAD

**Issue by debanjum** [khoj-ai/khoj#1021]
### Background
Khoj currently supports ability to copy chat messages. But you cannot easily copy references used by Khoj to generate responses. 

### Feature
- Add a copy references button to top of the references pane which opens when the "View references" link is clicked at the bottom of each Khoj chat response.
- This should copy all the reference web page and file path links as a markdown bullet list. For example:
   ```
   - [2024 - Wikipedia](https://en.wikipedia.org/wiki/2024)
   - [2023 - Wikipedia](https://en.wikipedia.org/wiki/2023)
   ```

**Comment by sanchitbansal10** [comment]
hi @debanjum , I can take this up. Please assign. Thanks

**Comment by debanjum** [comment]
Thanks for taking this up @sanchitbansal10 ! Let me know if you need any clarifications or have any questions

**Comment by adi790uu** [comment]
Hey @debanjum would like to work on this, please assign me this issue!

**Comment by debanjum** [comment]
Hey @adi790uu thanks for the interest! Let's wait a day for @sanchitbansal10 to respond/confirm?

@sanchitbansal10 is it fine if @adi790uu takes up this work item or are you still planning to work on it?

**Comment by adi790uu** [comment]
@debanjum I believe he is not active

**Comment by debanjum** [comment]
Ok, I've assigned it to you @adi790uu . Let us know if you need any help

**Comment by kowshik24** [comment]
@adi790uu are you still working on it?> Ok, I've assigned it to you @adi790uu . Let us know if you need any help

**Comment by debanjum** [comment]
@kowshik24 thanks for checking! Let me know if you want to work on this?

**Comment by adi790uu** [comment]
@kowshik24 yeah you can work on it, I was not able to work on this because of lack of time @debanjum please unassign me and sorry for not being able to work on this.

**Comment by debanjum** [comment]
No issues, thanks for letting us know!
